lib/matchers/languages.ml

```diff
@@ -690,7 +690,17 @@ module Julia = struct
 
   module Syntax = struct
     include Generic.Syntax
-
+    
+    Dyck.Syntax.user_defined_delimiters @
```

Comment:
Okay, fixed it now. 

By the way, Julia also uses 
```
module Foo

...

end # module
```
Does that cause an issue with these changes? I don't think it would be a good idea to add `module` to the user defined delimiters because then the entire "entrance" file for all packages will be considered by Comby to not be at the top level. This would unnecessary slow things down, I suppose. An example of a package where all the code is inside the module is https://github.com/JuliaCollections/Memoize.jl/blob/master/src/Memoize.jl. An example where most of the code is outside the module is https://github.com/JuliaData/DataFrames.jl.

Reply:
> I can't update your PR directly

That's weird. The "Allow edits by maintainers" box is checked.